vsts/build-release-artifacts.yml

```diff
@@ -1,9 +1,10 @@
 # update .csproj file versions and create a release branch/commit
 name: BumpVersion_$(BuildID)_$(Date:yyyyMMdd)$(Rev:.r)
+pool:
```

Comment:
The changes that are outside of the added task were validation errors exposed by ADO when testing this new build.